cli/src/pcluster/config/cluster_config.py

```diff
@@ -1699,23 +1654,10 @@ def _instance_type_info(self) -> InstanceTypeInfo:
             self.__instance_type_info = AWSApi.instance().ec2.get_instance_type_info(self.instance_type)
         return self.__instance_type_info
 
-    @property
-    def disable_simultaneous_multithreading_via_cpu_options(self) -> bool:
-        """Return true if simultaneous multithreading must be disabled through cpu options."""
-        return (
-            self.disable_simultaneous_multithreading
-            and self.instance_type_info.default_threads_per_core() > 1
-            and self.instance_type_info.is_cpu_options_supported_in_lt()
-        )
-
     @property
     def disable_simultaneous_multithreading_manually(self) -> bool:
         """Return true if simultaneous multithreading must be disabled with a cookbook script."""
-        return (
-            self.disable_simultaneous_multithreading
-            and self.instance_type_info.default_threads_per_core() > 1
-            and not self.instance_type_info.is_cpu_options_supported_in_lt()
-        )
+        return self.disable_simultaneous_multithreading and self.instance_type_info.default_threads_per_core() > 1
```

Comment:
I just noticed we had a lot of common methods for `HeadNode`  and `BaseComputeResource`. 
We should have them in a common parent class (we can think to this separately)

Reply:
I will leave this for a separate refactoring